<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerDialog</class>
 <widget class="QDialog" name="CustomerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customer Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Customer Details</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="cnicLabel">
       <property name="text">
        <string>CNIC:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="cnicEdit">
       <property name="placeholderText">
        <string>Enter CNIC without dashes</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="genderLabel">
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="genderComboBox">
       <item>
        <property name="text">
         <string>Male</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Female</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="phoneLabel">
       <property name="text">
        <string>Phone:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="phoneEdit">
       <property name="placeholderText">
        <string>e.g. 03001234567</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="addressEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save Details</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>